### DISCUSSION TITLE
Improve UI for One Sided Offers

### DISCUSSION THREAD

**Discussion by BrandtH22** [Chia-Network/chia-blockchain-gui#2417] [Ideas]
The current UI looks a bit odd for one sided offers, the below example is from a warp.green offer file received via walletConnect:
![image](https://github.com/Chia-Network/chia-blockchain-gui/assets/48610606/06c00a0f-943b-426d-a804-3432222c69ae)

**Comment by SlowestTimelord** [comment]
The confusing part to me is it sounds like you're (in this example) exchanging 1.001 XCH for "Fees".

I suppose there's a use case for a one-sided offer where the recipient is expected to include transaction fees, but then it really isn't one-sided and wouldn't be supported by warp.green anyway.

For a one-sided offer, maybe the right side can be replaced with a message that clearly explains that this offer is one-sided with a big "WARNING" that the offer file contents itself should *not* be shared or uploaded to any service except the service that initiated this Offer -- with the understanding that otherwise anyone that sees it will be able to take it.

**Comment by Ganbin** [comment]
I also think that the "empty" Fees box is confusing. If fees are not included, it should either :
1. Not showing an "empty" Fees box.
2. Showing an explicit message "No fees are included".

This would also make it clearer for offers that have assets on the other side. I would personally prefer option 1 as option 2 can also be confusing because we can still include fees ourselves.

For the one-sided offer, I think a clear message that this offer request nothing in exchange and should be accepted with caution and only with trusted platform. Eventually, it should also say that the offered assets can be drained if shared with untrusted third party.
